Add `The Followpoint: Flaro, the Queen of Speed` news post (#11832)
ppy/osu-wiki: news/2024/2024-07-29-the-followpoint-flaro-the-queen-of-speed.md

    }

    .news-chat-quote__username {
        font-weight: 600;
        margin-bottom: 2px;
    }

    .news-chat-quote__colour-lvd {
        color: #FFD1DC;
    }

    .news-chat-quote__colour-no-group {
        color: #FFFFFF;
    }
</style>

<div class="news-chat-quote__text-container">

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/11354436" style="background-image: url('/wiki/shared/news/2024-07-29-the-followpoint-flaro-the-queen-of-speed/avatar-Utiba.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/11354436">Utiba</a></p>

Let's start with an introduction, how would you introduce yourself to the people who don't know you?

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/10323184" style="background-image: url('/wiki/shared/news/2024-07-29-the-followpoint-flaro-the-queen-of-speed/avatar-Flaro.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/10323184">Flaro</a></p>

My name is Flaro, and I am a former #1 USA osu! player in both the United States as well as the NA region.

> I specialize in high bpm flow aim as well as 3 mod consistency, while also playing any tournaments that I happen to get invited to, where I specialize in mechanics heavy maps.

Reviewer: ```suggestion
I specialize in high BPM flow aim as well as 3-mod consistency, while also playing any tournaments that I happen to get invited to, where I specialize in mechanics heavy maps.
```

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/11354436" style="background-image: url('/wiki/shared/news/2024-07-29-the-followpoint-flaro-the-queen-of-speed/avatar-Utiba.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/11354436">Utiba</a></p>

How did you get introduced to osu!, and what was your first impression of it?

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/10323184" style="background-image: url('/wiki/shared/news/2024-07-29-the-followpoint-flaro-the-queen-of-speed/avatar-Flaro.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/10323184">Flaro</a></p>

> Shortly before my 11th birthday, I was introduced to the game by a friend of mine that I met through discord, at the time, the game didn't really captivate me that much and I stopped playing up until I turned 12, which is when I would start to play the game very actively and thought the game was very fun!

Reviewer: ```suggestion
Shortly before my 11th birthday, I was introduced to the game by a friend of mine that I met through Discord, at the time, the game didn't really captivate me that much and I stopped playing up until I turned 12, which is when I would start to play the game very actively and thought the game was very fun!
```

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/11354436">Utiba</a></p>

What made you come up with the name "Flaro", and is there any special meaning behind it?

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/10323184" style="background-image: url('/wiki/shared/news/2024-07-29-the-followpoint-flaro-the-queen-of-speed/avatar-Flaro.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/10323184">Flaro</a></p>

Although the name commonly gets misconstrued as a spin on the pokemon 'Flareon', the name Flaro actually just came to me while I was laying in bed one night and I liked it more than my previous name.

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/11354436" style="background-image: url('/wiki/shared/news/2024-07-29-the-followpoint-flaro-the-queen-of-speed/avatar-Utiba.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/11354436">Utiba</a></p>

> Out of all of the skillsets in the game, what made you want to prioritize speed?

Reviewer 2: ```suggestion
Out of all the skill sets in the game, what made you want to prioritize speed?
```

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/10323184" style="background-image: url('/wiki/shared/news/2024-07-29-the-followpoint-flaro-the-queen-of-speed/avatar-Flaro.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/10323184">Flaro</a></p>

> Funnily enough, I was exclusively a low bpm flow aim and jump player up until only mid 2022, which is when I would start to grind speed, going for a score on the one and only Lionheart map. It was when I skipped 800 in a Walmart whilst wearing a Megumin cosplay (true story!) that I decided to fully commit to getting better at speed and flow aim.

Reviewer: ```suggestion
Funnily enough, I was exclusively a low BPM flow aim and jump player up until only mid-2022, which is when I would start to grind speed, going for a score on the one and only [Lionheart](https://osu.ppy.sh/beatmapsets/1390922) map. It was when I skipped 800 in a Walmart whilst wearing a Megumin cosplay (true story!) that I decided to fully commit to getting better at speed and flow aim.
```

Reviewer 2: was there not a follow up question to the megumin cosplay in a walmart? i feel like thats something that will peak players' interest but its completely left alone?

Author: Asked Flaro a follow up question to this, did the same with the "It says in your Discord bio that you are a Python Developer, how did you get into coding in Python, and what do you like most about it?" question as well.

Currently going to add the suggested changes for now with the other consistency changes I have and add the follow up questions when I get a response.

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/11354436" style="background-image: url('/wiki/shared/news/2024-07-29-the-followpoint-flaro-the-queen-of-speed/avatar-Utiba.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/11354436">Utiba</a></p>

Do you feel like speed is really overweighted, why or why not?

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/10323184" style="background-image: url('/wiki/shared/news/2024-07-29-the-followpoint-flaro-the-queen-of-speed/avatar-Flaro.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/10323184">Flaro</a></p>

> Oh yeah definitely. Speed is in a very unbalanced state right now and I'm hopeful for the upcoming reworks that seem to aim to help with that issue, such as Xexxar's rework. I really hope to see a meta that's more focused on high bpm aim than anything as that skillset is very fun to me!

Reviewer: ```suggestion
Oh yeah definitely. Speed is in a very unbalanced state right now and I'm hopeful for the upcoming reworks that seem to aim to help with that issue, such as [Xexxar's rework](https://github.com/ppy/osu/discussions/20210). I really hope to see a meta that's more focused on high BPM aim than anything as that skillset is very fun to me!b
```

Reviewer 2: ```suggestion
Oh yeah definitely. Speed is in a very unbalanced state right now, and I'm hopeful for the upcoming reworks that seem to aim to help with that issue, such as [Xexxar's rework](https://github.com/ppy/osu/discussions/20210). I really hope to see a meta that's more focused on high bpm aim than anything as that skill set is very fun to me!
```

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/11354436">Utiba</a></p>

What are some things you could say that are unique to you compared to other top players?

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/10323184" style="background-image: url('/wiki/shared/news/2024-07-29-the-followpoint-flaro-the-queen-of-speed/avatar-Flaro.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/10323184">Flaro</a></p>

I would have to go with my 3 mod reading ability and consistency, and also my flow aim capabilities, as I have set many unmatched scores to this day on maps of those categories.

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/11354436" style="background-image: url('/wiki/shared/news/2024-07-29-the-followpoint-flaro-the-queen-of-speed/avatar-Utiba.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/11354436">Utiba</a></p>

> It says in your Discord bio that you are a Python Developer, how did you get into coding in Python and what do you like most about it?

Reviewer 2: ```suggestion
It says in your Discord bio that you are a Python Developer, how did you get into coding in Python, and what do you like most about it?
```

I've had a passion for technology for most of my life, just messing around with computers and stuff. I first started learning Python back in 2022 when I started making my own discord bots, and now I just program things for fun! I love that you can basically do anything you want when it comes to programming your own things. I'm currently in college for CompSci and own a server in my home as well.

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/11354436" style="background-image: url('/wiki/shared/news/2024-07-29-the-followpoint-flaro-the-queen-of-speed/avatar-Utiba.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/11354436">Utiba</a></p>

What made you change your name to "Leah Kate" for a short period of time?

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/10323184" style="background-image: url('/wiki/shared/news/2024-07-29-the-followpoint-flaro-the-queen-of-speed/avatar-Flaro.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/10323184">Flaro</a></p>

> There wasn't really a specific reason to it other than that I thought it'd be funny and I was popping off on the notorious Leah Kate farm maps at the time. Well worth the name change as now I have a 'Leah Kate arc' in my history!

Reviewer 2: ```suggestion
There wasn't really a specific reason to it other than that I thought it'd be funny, and I was popping off on the notorious Leah Kate farm maps at the time. Well worth the name change as now I have a 'Leah Kate arc' in my history!
```

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/11354436" style="background-image: url('/wiki/shared/news/2024-07-29-the-followpoint-flaro-the-queen-of-speed/avatar-Utiba.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/11354436">Utiba</a></p>

In previous months, you've had many pp record chokes such as a [Glory Days 3 mod S rank](https://osu.ppy.sh/scores/1719644015) and a [Lionheart 3 mod 1 miss](https://osu.ppy.sh/scores/1727333870). Have those near-misses evoked any particular feelings?

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/10323184" style="background-image: url('/wiki/shared/news/2024-07-29-the-followpoint-flaro-the-queen-of-speed/avatar-Flaro.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/10323184">Flaro</a></p>

> It can be really demotivating to be so so close to a goal but just one singular mistake costs it from you. Although I will say, the pp record grind had some of the most rewarding feelings I have ever felt while playing this game. Being close to getting a highly contested world record feels amazing, and even though I didn't get it, it was well worth the grind.

Reviewer 2: ```suggestion
It can be really demotivating to be so, so close to a goal, but just one singular mistake costs it from you. Although I will say, the pp record grind had some of the most rewarding feelings I have ever felt while playing this game. Being close to getting a highly contested world record feels amazing, and even though I didn't get it, it was well worth the grind.
```

I wasn't even really interested in getting #1 USA until the day that I got it, one of my biggest pop off days to date. It was on that day where the drive to become the best player in the USA really kept me going and pushed me to farm and farm and farm. Even after getting #1 USA I kept pushing, peaking at rank #7.

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/11354436" style="background-image: url('/wiki/shared/news/2024-07-29-the-followpoint-flaro-the-queen-of-speed/avatar-Utiba.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/11354436">Utiba</a></p>

Is there any specific scores you want to achieve or see others achieve in the future?

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/10323184" style="background-image: url('/wiki/shared/news/2024-07-29-the-followpoint-flaro-the-queen-of-speed/avatar-Flaro.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/10323184">Flaro</a></p>

> Scores that are on my to-do list currently are probably [Rat Race](https://osu.ppy.sh/beatmapsets/958005#osu/2069969), [Everything went Numb](https://osu.ppy.sh/beatmapsets/414289#osu/898106), [Happy! Lucky!](https://osu.ppy.sh/beatmapsets/1594964#osu/3257742), and just focusing on more aim oriented scores in general. I don't really pay attention to the scores that others get, but I always get super hyped when I see another high bpm aim score from [mrekk](https://osu.ppy.sh/users/7562902), those are always legendary to watch and I hope to see more.

Reviewer 2: ```suggestion
Scores that are on my to-do list currently are probably [Rat Race](https://osu.ppy.sh/beatmapsets/958005#osu/2069969), [Everything Went Numb](https://osu.ppy.sh/beatmapsets/414289#osu/898106), [Happy! Lucky!](https://osu.ppy.sh/beatmapsets/1594964#osu/3257742), and just focusing on more aim oriented scores in general. I don't really pay attention to the scores that others get, but I always get super hyped when I see another high bpm aim score from [mrekk](https://osu.ppy.sh/users/7562902), those are always legendary to watch, and I hope to see more.
```

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/11354436" style="background-image: url('/wiki/shared/news/2024-07-29-the-followpoint-flaro-the-queen-of-speed/avatar-Utiba.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/11354436">Utiba</a></p>

To finish things up, what does the future hold for you? Do you have any goals or anything else that you'd like to mention?

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/10323184" style="background-image: url('/wiki/shared/news/2024-07-29-the-followpoint-flaro-the-queen-of-speed/avatar-Flaro.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/10323184">Flaro</a></p>

> I will continue the osu! grind until my motivation to do so declines, although as of recent my focus has been much less on pp and more on tourney performances and just playing for fun! Although attempting to reclaim #1 USA isn’t completely out of the question, it’s unlikely unless my interest in grinding for pp comes back. In terms of goals, my biggest goal at the moment is definitely getting better at tournaments and winning OWC, and being crowned an osu! champion, along with that, I want to focus more on my growth as a [streamer on Twitch](https://www.twitch.tv/flarosu), and would love to get invited to Roundtable sometime in the future, meeting other streamers like me always makes me happy!

Reviewer 3: ```suggestion
I will continue the osu! grind until my motivation to do so declines, although as of recent my focus has been much less on pp and more on tourney performances and just playing for fun! Although attempting to reclaim #1 USA isn’t completely out of the question, it’s unlikely unless my interest in grinding for pp comes back. In terms of goals, my biggest goal at the moment is definitely getting better at tournaments and winning OWC, and being crowned an osu! champion, along with that, I want to focus more on my growth as a [streamer on Twitch](https://www.twitch.tv/flarosu), and would love to get invited to The Roundtable sometime in the future, meeting other streamers like me always makes me happy!
```

Reviewer 2: this entire block of text needs to be broken up into 1 or 2 paragraphs

---
layout: post
title: "The Followpoint: Flaro, the Queen of Speed"
date: 2024-07-29 17:00:00 +0000
series: The Followpoint
---

> Wake up babe, a new interview with [Flaro](https://osu.ppy.sh/users/10323184) just dropped.

Reviewer 4: i disagree with putting a meme as the preview

Author: Do you have any suggestions on what this should be?

Author: Adding onto this, I don't see an issue why it shouldn't be a meme. I see a couple other newsposts have made their previews as memes.

Reviewer 4: if it fits the tone we generally go for. they also usually give an idea of what's in the article

```suggestion
A new top player interview just dropped. This time: speed meta opinions and the cosplay buff.
```

Author: Fair enough I guess

![](/wiki/shared/news/2024-07-29-the-followpoint-flaro-the-queen-of-speed/banner.jpg)

> Welcome back to The Followpoint! Today, we are joined by Flaro, a top speed player who was #1 USA for many months at a time and having numerous pp record chokes that could of changed the game forever!

Reviewer 4: ```suggestion
Welcome back to The Followpoint! Today, we are joined by Flaro, a top speed player who has been #1 in USA for many months at a time and at several points nearly (but not quite) broken the performance point record!
```

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/10323184">Flaro</a></p>

My name is Flaro, and I am a former #1 USA osu! player in both the United States as well as the NA region.

I specialize in high BPM flow aim as well as 3-mod consistency, while also playing any tournaments that I happen to get invited to, where I specialize in mechanics heavy maps.

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/11354436" style="background-image: url('/wiki/shared/news/2024-07-29-the-followpoint-flaro-the-queen-of-speed/avatar-Utiba.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/11354436">Utiba</a></p>

How did you get introduced to osu!, and what was your first impression of it?

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/10323184" style="background-image: url('/wiki/shared/news/2024-07-29-the-followpoint-flaro-the-queen-of-speed/avatar-Flaro.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/10323184">Flaro</a></p>

> Shortly before my 11th birthday, I was introduced to the game by a friend of mine that I met through Discord, at the time, the game didn't really captivate me that much and I stopped playing up until I turned 12, which is when I would start to play the game very actively and thought the game was very fun!

Reviewer 4: comma splice

```suggestion
Shortly before my 11th birthday, I was introduced to the game by a friend of mine that I met through Discord. At the time, the game didn't really captivate me that much and I stopped playing up until I turned 12, which is when I would start to play the game very actively and thought the game was very fun!
```

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/11354436" style="background-image: url('/wiki/shared/news/2024-07-29-the-followpoint-flaro-the-queen-of-speed/avatar-Utiba.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/11354436">Utiba</a></p>

To finish things up, what does the future hold for you? Do you have any goals or anything else that you'd like to mention?

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/10323184" style="background-image: url('/wiki/shared/news/2024-07-29-the-followpoint-flaro-the-queen-of-speed/avatar-Flaro.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/10323184">Flaro</a></p>

I will continue the osu! grind until my motivation to do so declines, although as of recent my focus has been much less on pp and more on tourney performances and just playing for fun! Although attempting to reclaim #1 USA isn’t completely out of the question, it’s unlikely unless my interest in grinding for pp comes back.

> In terms of goals, my biggest goal at the moment is definitely getting better at tournaments and winning OWC, and being crowned an osu! champion, along with that, I want to focus more on my growth as a [streamer on Twitch](https://www.twitch.tv/flarosu), and would love to get invited to The Roundtable sometime in the future, meeting other streamers like me always makes me happy!

Reviewer 4: same here

```suggestion
In terms of goals, my biggest goal at the moment is definitely getting better at tournaments and winning OWC, and being crowned an osu! champion. Along with that, I want to focus more on my growth as a [streamer on Twitch](https://www.twitch.tv/flarosu), and would love to get invited to The Roundtable sometime in the future. Meeting other streamers like me always makes me happy!
```
